jenkins/deploy.sh

```diff
@@ -125,7 +125,7 @@ $DEPLOY_CMD -DpomFile=$POM_FILE \
 
 echo "$ART_GROUP_ID:$ART_ID:$ART_VER:jar" >> $ARTIFACT_FILE
 CLASSLIST="$CLASSIFIERS,sources,javadoc"
-CLASSLIST=(${CLASSLIST//','/' '})
-for class in ${CLASSLIST[@]}; do
+CLASSLIST=("${CLASSLIST//','/' '}")
+for class in "${CLASSLIST[@]}"; do
```

Comment:
This `for class in "${CLASSLIST[@]}"; do` will makes `cuda12 cuda12-arm64 sources javadoc` as one string instead of for array items

We should use `for class in ${CLASSLIST[@]}; do` instead.

Let's revert this change first, we may need more detailed verify on the change, thanks